exampleSite/config/_default/config.toml

```diff
@@ -23,6 +23,7 @@ googleAnalytics = ""
   show_events = true
   table_classes = "table table-bordered"
   hide_cfa_same_page = true
+  # hide_breadcrumb = true
```

Comment:
If this is not needed, I would remove it from the PR. Or maybe you'd like to add this line there to let people know of the available options?

Reply:
Yes, I want to put there as an available option

Reply:
When we have an available option, we should document it on the site, not in the actual files. In this case, we could document this on the homepage, or in the breadcrumb component page.